ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml

```diff
@@ -103,10 +103,24 @@ SSS:
   is_abstract: true
   implements:
   - /SENSOR
-
+  
 HDS:
   guid: "025f5773-7626-4a30-bd44-ef0845a92215"
   description: "Hydrogen Sensor."
   is_abstract: true
   implements:
   - /SENSOR
+
+EHT:
+  guid: "700c69bb-8d37-4e5f-806f-ae64d007cf35"
+  description: "Heat Tracer."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+MFSD:
```

Comment:
Remove this and create something like `FD_MDC`

Reply:
I see, will do.